src/cli/DownloadAndProcessInTransactionCommand.ts

```diff
@@ -0,0 +1,35 @@
+import { CLICommand } from "./CLICommand";
+import { DatabaseConnection } from "../database/DatabaseConnection";
+import { ImportFeedInTransactionCommand } from './ImportFeedCommandWithFallback';
+
+export class DownloadAndProcessInTransactionCommand implements CLICommand {
```

Comment:
When other commands will be move to "transactional" interface, we can reuse this.